exercises/triangle/canonical-data.json

```diff
@@ -184,13 +184,22 @@
         },
         {
           "uuid": "c6e15a92-90d9-4fb3-90a2-eef64f8d3e1e",
-          "description": "two sides are equal",
+          "description": "two adjacent sides are equal",
           "property": "scalene",
           "input": {
             "sides": [4, 4, 3]
           },
           "expected": false
         },
+        {
+          "uuid": "3da23a91-a166-419a-9abf-baf4868fd985",
+          "description": "two unadjacent sides are equal",
```

Comment:
Calling the sides unadjacent makes me uncomfortable. Since it's a triangle, for any pair of sides, those two sides are adjacent to each other (they share a vertex). This word "adjacent" is therefore used in contravention of usual meaning, because here it was used to mean whether the two numbers are adjacent in the input list.

However, I have recognised this as a situation where I cannot be allowed to let my discomfort stand in the way of progress, which is why I Approved anyway.

Reply:
True, I didn't think about that. Any suggestions on how to better name the description?

Reply:
Referring to the sides as first, second, and third could work, since I don't recall the possibility of these three words being misunderstood to mean something, unlike "adjacent". Of course someone might fairly complain that then a case of the form `[a, b, b]` is missing. Your choice whether to add that `[a, b, b]` case or not.

Reply:
Sure, that will work, I think I will just change it to "first and second sides are equal" and "first and third sides are equal".

Reply:
> Your choice whether to add that `[a, b, b]` case or not.

If it were up to me, this case would be added, just in case (and because it is cheap).

The code that slipped through the tests originally went along the lines of (Python) `a, b, c = sides ; return a != b != c`. A variant that might still slip through is `b != a != c`.

Reply:
@MatthijsBlom On a second thought, yep, it's best to add that as well.